api/config/cdr_config_preprod.json

```diff
@@ -103,6 +102,23 @@
       "hasFitbitData": true,
       "hasCopeSurveyData": true,
       "accessTier": "registered"
+    },
+    {
+      "cdrVersionId": 7,
+      "name": "All of Us Controlled Tier Dataset v6 beta",
+      "isDefault": true,
+      "bigqueryProject": "fc-aou-cdr-preprod-ct",
+      "bigqueryDataset": "C2021Q3R2",
+      "wgsBigqueryDataset": "alpha2_prelim2",
```

Comment:
Sorry for the delay - I was checking on the status of this. Please just omit these fields for now:
- wgsBigqueryDataset
- allSamplesWgsDataBucket
- singleSampleArrayDataBucket

the genomics data is not ready to push into preprod yet

Reply:
done.